play_motion2_executor:
  ros__parameters:
    motion_planner:
      approach_velocity: 0.5
      approach_min_duration: 0.5

